lib/ansible/modules/database/postgresql/postgresql_db.py

```diff
@@ -201,6 +201,11 @@
 class NotSupportedError(Exception):
     pass
 
+
+class LibraryError(Exception):
```

Comment:
@Andersson007 Are you OK with this exception ? @sbulage Could you please add a change log entry [here](https://github.com/ansible/ansible/tree/devel/changelogs/fragments) ?

Reply:
@Akasurde Added change log entry. Thanks for quick review.

Reply:
@Akasurde , thanks for mentionint!
i left a comment, it's not needed, we can just remove the problem lines from postgresql_db, see the comment below